.github/workflows/docker.yml

```diff
@@ -1,24 +1,67 @@
 name: Docker
 
 on:
-  pull_request:
-    branches:
-      - 'main'
-    paths:
-      - '.github/workflows/docker.yaml'
-      - 'dependencies/current/'
-      - 'dependencies/testing/'
+  workflow_dispatch:
+  # pull_request:
+  #   branches:
+  #     - 'main'
+  #   paths:
+  #     - '.github/workflows/docker.yml'
+  #     - 'dependencies/current/'
+  #     - 'dependencies/testing/'
```

Comment:
```suggestion
```

Reply:
I removed the TODOs. The docker.yml needs a final overhaul anyway.